src/sentry/static/sentry/app/components/repositoryFileSummary.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import FileChange from './fileChange';
+import {tn} from '../locale';
+
+function Collapsed(props) {
+  return (
+    <li className="list-group-item list-group-item-sm align-center">
+      <span className="icon-container">
+      </span>
+      <a onClick={props.onClick}>{tn(('Show ' + props.count + ' collapsed file'), ('Show ' + props.count + ' collapsed files'), props.count)}</a>
+    </li>
+  );
+}
+
+Collapsed.propTypes = {
+  onClick: React.PropTypes.func.isRequired,
+  count: React.PropTypes.number.isRequired
+};
+
+const RepositoryFileSummary = React.createClass({
+  propTypes: {
+    fileChangeSummary: React.PropTypes.object,
+    repository: React.PropTypes.string,
+  },
+
+  statics: {
+    MAX_WHEN_COLLAPSED: 5
+  },
+
+  getInitialState() {
+    return {
+      loading: true,
+      collapsed: true,
+    };
+  },
+
+  onCollapseToggle() {
+    this.setState({
+      collapsed: !this.state.collapsed
+    });
+  },
+
+  render() {
+    let {repository, fileChangeSummary} = this.props;
+    const MAX = RepositoryFileSummary.MAX_WHEN_COLLAPSED;
+    let files = Object.keys(fileChangeSummary);
+    let fileCount = files.length;
+    files.sort();
+    if (this.state.collapsed && fileCount > MAX) {
+      files = files.slice(0, MAX);
+    }
+    let numCollapsed = fileCount - files.length;
+    let canCollapse = fileCount > MAX;
+    return(
+      <ul className="list-group list-group-striped m-b-2">
+      <h6>
+        {fileCount} {tn((' file changed in ' + repository), (' files changed in ' + repository), fileCount)}
```

Comment:
same here, i think this can just be
`{tn(('%d file changed in ' + repository), ('%d files changed in ' + repository), fileCount)}`